static/main.ts

```diff
@@ -461,48 +478,42 @@ function removeOrphanedMaximisedItemFromConfig(config) {
 
     impl(config);
 
-    if (!found) {
+    if (!(found as boolean)) {
```

Comment:
There are shenanigans at play here. TS has narrowed `found` to `false`, it doesn't seem to understand the closure.